include/micm/configure/solver_config.hpp

```diff
@@ -72,16 +73,19 @@ namespace micm
   {
     System system_;
     std::vector<Process> processes_;
+    RosenbrockSolverParameters parameters_;
 
-    SolverParameters(const System& system, std::vector<Process>&& processes)
+    SolverParameters(const System& system, std::vector<Process>&& processes, const RosenbrockSolverParameters&& parameters)
         : system_(system),
-          processes_(std::move(processes))
+          processes_(processes),
+          parameters_(parameters)
```

Comment:
My understanding is that `std::move` is necessary to actually move what `processes` owns to `processes_`. Could you confirm that it's not copy?

Reply:
It isn't necessary here. `std::move` only attempts to create an rvalue reference, but it doesn't actually move anything. If we are in this constructor, then `parameters` is already an rvalue reference (`&&`).